backend/app/controllers/concerns/transaction_handler.rb

```
@@ -7,7 +7,15 @@ def start_transaction_and_rollback_on_exception
         ActiveRecord::Base.transaction do
             yield
         rescue StandardError => e
-            render_error(message: e.message)
+            begin
+                render_error(message: e.message)
+            rescue NoMethodError
```

Is this the only error that can happen? I would leave it as rescue from StandardError and let it bubble up.